src/main/java/User/Methods/Methods.java

```diff
@@ -0,0 +1,7 @@
+package User.Methods;
```

Comment:
non comprendo l'utilità di questo package con dentro una sola classe. Questi sono i ruoli dello user, dovrebbero stare nel package user. Il package serve in teoria se vanno raggruppate più cose, non per una sola classe